app/models/agents/survey_monkey_agent.rb

```diff
@@ -169,6 +166,17 @@ def per_page
       interpolated['per_page']
     end
 
+    def build_survey(survey_id)
+      survey = fetch_survey_details(survey_id)
+      survey['responses'] = fetch_survey_responses(survey_id)
+      if boolify(interpolated['guess_mode']) == false
```

Comment:
maybe just unless?

Reply:
@OpakAlex boolify can return false/nil, in this case needs to be explicitly false